tests/integ/scala/test_function_suite.py

```diff
@@ -3835,6 +3835,52 @@ def test_convert_timezone(session, local_testing_mode):
         LocalTimezone.set_local_timezone()
 
 
+def test_convert_timezone_neg(session):
+    df = TestData.datetime_primitives1(session)
+    with pytest.raises(SnowparkSQLException):
+        df.select(
+            convert_timezone(lit("UTC"), "timestamp_tz", lit("US/Eastern"))
+        ).collect()
+
+
+def test_convert_timezone_nulls(session):
+    null_df = session.create_dataframe([[None]]).to_df("timestamp")
```

Comment:
dumb question, do we have test for non-None two args input for convert_timezone?

Reply:
Yes, but I've left that one unchanged.